src/index.html

```diff
@@ -1,12 +1,16 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>CSS Blocks - webpack</title>
+  <link rel="stylesheet" href="./blocks.css">
```

Comment:
I guess this is temporal, in the future CSSBlock plugin would extract/emit them or maybe be compatible with https://github.com/webpack-contrib/mini-css-extract-plugin

Reply:
yes, we need to get that sorted out.